rotkehlchen/data_migrations/migrations/migration_11.py

```diff
@@ -42,10 +41,49 @@ def _get_optimism_transaction_fees(rotki: 'Rotkehlchen', progress_handler: 'Migr
         )
 
 
+def _get_optimism_transaction_fees_num_steps(rotki: 'Rotkehlchen') -> int:
+    """Returns the number of progress handler steps needed for the get optimism transaction fees"""
+    with rotki.data.db.conn.read_ctx() as cursor:
+        cursor.execute('SELECT tx_hash, identifier FROM evm_transactions WHERE chain_id=10')
+        txhash_to_id = dict(cursor)
+    return len(txhash_to_id)
+
+
+def _detect_arbitrum_one_accounts_num_steps(rotki: 'Rotkehlchen') -> int:
+    """Returns the number of progress handler steps needed for the detect arbitrum one accounts"""
+    with rotki.data.db.conn.read_ctx() as cursor:
+        accounts = rotki.data.db.get_blockchain_accounts(cursor)
+    # steps are: ethereum accounts + potentially write to db + updating spam assets
+    return len(accounts.eth) + 3
+
+
+def _detect_arbitrum_one_accounts(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:  # noqa: E501
+    """It duplicates accounts if active at arbitrum one only. The reason is to start the new
+    release with arbitrum one already detected"""
```

Comment:
This is not what this function does. you are actually detecting **all** evm chain accounts. This runs for arbitrum, polygon, optimism, avalanche. Which is what the last release migration also did, minus the arbitrum. "Duplication" is not the right word here.

```suggestion
    """Detect accounts that have activity in other evm chains and are not yet tracked. The reason is to start the new
    release with accounts active at arbitrum already detected"""
```

Reply:
this hasn't be done

Reply:
Also can you comment that first spam assets and rpc nodes are updated?